soc/arm/renesas_smartbond/da1469x/soc.h

```diff
@@ -13,6 +13,11 @@ extern "C" {
 
 #include <DA1469xAB.h>
 
+int z_smartbond_sleep(void);
+void z_smartbond_wakeup_handler(void);
+void z_smartbond_wakeup(void);
+void z_smartbond_restore_state(void);
```

Comment:
please do not pollute soc.h which is included almost everywhere (due to CMSIS design), create a dedicated header.

Reply:
moved to smartbond_sleep.h